opencsp/common/lib/deflectometry/BlobIndex.py

```diff
@@ -12,6 +12,18 @@ class BlobIndex:
     X/Y axes correspond to image axes; +x is to right, +y is down. Class takes in points (in units
     of pixels) that have been previously found with a blob detector and attempts to assign all found
     xy pixel points with a blob index.
+
+    Attributes
+    ----------
+    search_thresh : float
+
+    search_perp_axis_ratio : float
+        Ratio of point distances: (perpendicular to axis) / (along axis) used to
+        search for points.
+    apply_filter : bool
+        To filter bad points (experimental)
+    verbose : bool
+        Deprecated
```

Comment:
In general, when something is deprecated, it's a good idea to print a warning when it gets used.  Because verbose is a property, that means in the getter/setter.  For example:

```python
@property
def verbose(self):
    lt.warn("Warning: BlobIndex property 'verbose' is deprecated and will be removed in a future release. Use 'do_plot' instead")
    return self._verbose

@verbose.setter
def verbose(self, value: bool):
    lt.warn("Warning: BlobIndex property 'verbose' is deprecated and will be removed in a future release. Use 'do_plot' instead")
    self._verbose = value
```